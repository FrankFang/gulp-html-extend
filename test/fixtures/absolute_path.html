<!-- @@master=/fixtures/master_b.html {"var":"var in extend_and_include.html"}-->
<!-- @@block=main-->
<article>
    I am extend_and_include.html
    <!--@@include= /fixtures/include.html {"a":123456789}-->
</article>
<!-- @@close-->
