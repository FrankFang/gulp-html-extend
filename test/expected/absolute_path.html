<!DOCTYPE html>
<html lang="en">
    <head>
        <meta charset="UTF-8">
        <title>var in master_b.html</title>
<!-- start include.html-->
included content
987654321"
<!-- start include_include.html-->
included * 2

<!-- end include_include.html-->

<!-- end include.html-->
    </head>
    <body>
<!-- start content -->
<main>
    my content
    var in extend_and_include.html
<!-- start main -->
<article>
    I am extend_and_include.html
<!-- start include.html-->
included content123456789
987654321"
<!-- start include_include.html-->
included * 2

<!-- end include_include.html-->

<!-- end include.html-->
</article>

<!-- end main -->
</main>

<!-- end content -->
<!-- start footer -->
<footer>
    my footer
</footer>

<!-- end footer -->
    </body>
</html>
